packages/p2p-v2/src/components/PaymentMethodForm/PaymentMethodForm.tsx

```diff
@@ -95,6 +95,7 @@ const PaymentMethodForm = ({ onAdd, onResetFormState, ...rest }: TPaymentMethodF
                         <Input
                             disabled
                             label='Choose your payment method'
+                            onChange={() => undefined}
```

Comment:
This is to resolve:
<img width="1273" alt="Screenshot 2024-01-30 at 12 41 45 PM" src="https://github.com/binary-com/deriv-app/assets/103016120/7789a9bb-6900-4d59-8ba8-223a66e45a5f">

Reply:
this is weird, i think we need to bring this up with @shayan-binary regarding this

Reply:
i just checked it with @shayan-deriv you can use defaultValue instead of value and remove the onChange method.